src/components/Collection/SeriesMetadata.tsx

```diff
@@ -3,58 +3,47 @@ import { mdiCloseCircleOutline, mdiOpenInNew, mdiPencilCircleOutline, mdiPlusCir
 import { Icon } from '@mdi/react';
 
 import Button from '@/components/Input/Button';
-import { useDeleteSeriesTvdbLinkMutation } from '@/core/react-query/series/mutations';
+import { useDeleteSeriesTmdbLinkMutation, useDeleteSeriesTvdbLinkMutation } from '@/core/react-query/series/mutations';
 import useEventCallback from '@/hooks/useEventCallback';
 
-const MetadataLink = (
-  { id, seriesId, site, type }: {
-    id?: number | number[] | null;
-    seriesId: number;
-    site: 'AniDB' | 'TMDB' | 'TvDB' | 'TraktTv';
-    type?: 'Movie' | 'Show';
-  },
-) => {
-  const linkId = Array.isArray(id) ? id[0] || null : id ?? null;
+type Props = {
+  id?: number;
+  seriesId: number;
+  site: 'AniDB' | 'TMDB' | 'TvDB' | 'TraktTv';
+  type?: 'Movie' | 'Show';
+};
 
+const MetadataLink = ({ id, seriesId, site, type }: Props) => {
```

Comment:
> was refering to the name of the hook, not the local variable. 😔

Reply:
You should've mentioned it properly 😔 
Fixed it